core/base/src/TSystem.cxx

```diff
@@ -719,6 +719,7 @@ int TSystem::GetPid()
 void TSystem::Exit(int, Bool_t)
 {
    AbstractMethod("Exit");
+   throw; // unreachable
```

Comment:
@pcanal I think maybe we could actually throw in `TObject::AbstractMethod()`.  @pcanal What do you think?

Reply:
You mean something like?
`if(method == "Exit" || method == "Abort") throw;`
?

Reply:
I don't think the extra complexity is worth it.  Maybe @jalopezg-r00t meants to always throw in `AbstractMethod` but this would be a change in behavior (for all the other use of `AbstractMethod`)